ui-components/src/page_components/listing/listing_sidebar/LeasingAgent.tsx

```diff
@@ -35,7 +37,6 @@ const LeasingAgent = (props: LeasingAgentProps) => {
               {listing.leasingAgentPhone}
             </a>
           </p>
-          <p className="text-sm text-gray-700">{t("leasingAgent.dueToHighCallVolume")}</p>
```

Comment:
Could you try overriding this string to empty/null to see if that removes it? (or will the space still be there?) That way we wouldn't need any changes that can't be contributed upstream.

Reply:
That worked! Good idea :)